### DISCUSSION TITLE
Hosting Nextjs on windows server

### DISCUSSION THREAD

**Discussion by Moh-Nasser** [vercel/next.js#45135] [Help]
Hello,

I am new to nextjs and i am currently using it for building an ecommerce project, but i am struggling with hosting on my cloud windows server.
Usually in create react app i would run "npm build" and use iis to host the build files, but nextjs is somewhat different... I managed to run it using the "npm build" and "npm start" but its running on port 3000 and i want to run it on the server default https port so i can use www.mydomian.com instead of www.mydomain.com:3000.

Any help is appreciated

**Comment by vradriano** [comment]
On the package.json, in the start script you can use:
```javascript
  "scripts": {
    "start": "next start -p 80 --hostname mydomain.com"
  }
```

I'm not sure if it will help, but i hope so
